scripts/modules/elastic_stack.py

```diff
@@ -656,6 +656,11 @@ def __init__(self, **options):
                 self.environment["STATUS_ALLOWANONYMOUS"] = "true"
         self.environment["ELASTICSEARCH_URL"] = ",".join(self.options.get(
             "kibana_elasticsearch_urls") or [self.DEFAULT_ELASTICSEARCH_HOSTS])
+        if self.at_least_version("7.6") and options.get("xpack_secure"):
+            self.environment["XPACK_SECURITY_LOGINASSISTANCEMESSAGE"] = "Login\\_details:\\_`{}/{}`.\\_" \
```

Comment:
```suggestion
            self.environment["XPACK_SECURITY_LOGINASSISTANCEMESSAGE"] = "Login&#32;details:&#32;`{}/{}`.&#32;" \
```

Reply:
Let me try this proposal

Reply:
Changed -> https://github.com/elastic/apm-integration-testing/pull/702/commits/280e113546bc990d045a486c9ec0404437a34779